cms/templates/widgets/footer.html

```diff
@@ -19,18 +19,6 @@
           <li class="nav-item nav-peripheral-pp">
             <a data-rel="edx.org" href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
           </li>
-<<<<<<< HEAD
-          % if settings.TENDER_DOMAIN and user.is_authenticated():
-          <li class="nav-item nav-peripheral-feedback">
-            <a data-rel="edx.org" href="http://${settings.TENDER_DOMAIN}/discussion/new" class="show-tender" title="${_('Use our feedback tool, Tender, to share your feedback')}">${_("Contact Us")}</a>
-          </li>
-          % elif user.is_authenticated():
```

Comment:
We can put the contact email in the sock by setting `PARTNER_SUPPORT_EMAIL` in features